mopidy_musicbox_webclient/static/js/functionsvars.js

```diff
@@ -343,12 +343,9 @@ function resultsToTables(results, target, uri) {
                     }
                     newalbum = [];
                     newtlids = [];
-                    if (results[i].album) {
-                        coversList.push([results[i].album, i]);
-                    }
                 } //newalbum length
                 if (results[i].album) {
-                    coversList.push([results[i].album, i]);
+                    coversList.push([results[i].uri, i]);
```

Comment:
...so the idea is that we only push an album cover if more than one track is being shown for a particular album?

Done in 7d680fe252b5d4e5c28619e8ba1e67705891a0c2!